spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/DataSet.scala

```diff
@@ -355,6 +355,54 @@ object DataSet {
   }
 
   /**
+   * Wrap an array as RDD
+   * @param localData
+   * @param sc spark context
+   * @param partitonNum paritition number of rdd, default Engine.nodeNumber()*Engine.coreNumber()
+   */
+  def arrayToRdd[T: ClassTag](localData: Array[T],
+                         sc: SparkContext, partitonNum: Option[Int] = None): RDD[T] = {
+    val nodeNumber = Engine.nodeNumber()
+    val coreNumber = Engine.coreNumber()
+    val num = partitonNum.getOrElse(nodeNumber * coreNumber)
+    sc.parallelize(localData, num)
+  }
+
+  /**
+   * load sequence file to rdd
+   * @param url
+   * @param sc
+   * @param partitonNum paritition number of rdd, default Engine.nodeNumber()*Engine.coreNumber()
+   * @return
+   */
+  def sequenceToRdd(url: String, sc: SparkContext, partitonNum: Option[Int] = None)
+  : RDD[ByteRecord] = {
+    val nodeNumber = Engine.nodeNumber()
+    val coreNumber = Engine.coreNumber()
+    val num = partitonNum.getOrElse(nodeNumber * coreNumber)
+    sc.sequenceFile(url, classOf[Text], classOf[Text], num).map(image => {
+      ByteRecord(image._2.copyBytes(), DataSet.SeqFileFolder.readLabel(image._1).toFloat)
+    })
+  }
+
+  /**
+   * Apply transformer to rdd
+   * @param dataset
+   * @param transformer
+   */
+  def transform[C: ClassTag, D: ClassTag](dataset: RDD[C],
```

Comment:
Make it as a method of Transformer. Instead of
rdd2 = DataSet.transform(rdd1, transfomer)

why not
rdd2 = transformer(rdd1)?